server/src/main/java/com/strategists/game/controller/PlayerController.java

```diff
@@ -42,21 +34,21 @@ public List<Player> getPlayers() {
 	}
 
 	@PostMapping
-	public Player addPlayer(@RequestBody Map<String, Object> body) {
+	public Player addPlayer(@RequestBody AddPlayerRequest request) {
 		Assert.state(gameService.isLobbyState(), "Can't add players to active game!");
-		return playerService.addPlayer((String) body.get(Keys.USERNAME), Double.parseDouble(body.get(Keys.CASH).toString()));
+		return playerService.addPlayer(request.getUsername(), request.getCash());
```

Comment:
Have you been able to check if it is working with Integer value? TypeScript truncates the decimal value if it is 0

Reply:
POJO fixes that.